src/SonarScanner.MSBuild.PostProcessor/MSBuildPostProcessor.cs

```diff
@@ -85,18 +94,47 @@ public bool Execute(string[] args, AnalysisConfig config, ITeamBuildSettings set
                 return false;
             }
 
-            // if initialization fails a warning will have been logged at the source of the failure
-            var initialised = codeCoverageProcessor.Initialise(config, settings);
+            var propertyResult = GenerateAndValidatePropertiesFile(config);
 
-            if (initialised && !codeCoverageProcessor.ProcessCoverageReports())
+            if (propertyResult.RanToCompletion)
             {
-                // if processing fails, stop the workflow
-                return false;
+#if NET46
+                ProcessCoverageReport(config, Path.Combine(config.SonarConfigDir, "SonarQubeAnalysisConfig.xml"), propertyResult.FullPropertiesFilePath);
```

Comment:
There is a constant for `SonarQubeAnalysisConfig.xml` in `FileConstants.ConfigFileName`, which I think can be used here and in line 108.